src/api/pack/xdsl/move/pack-xdsl-move.v6.service.js

```diff
@@ -38,5 +43,33 @@ angular.module('ovh-api-services').service('OvhApiPackXdslMoveV6', ($resource, P
     );
   };
 
+  move.pollOffers = function ($scope, opts) {
+    const url = ['/pack/xdsl/', opts.packName, '/addressMove/offers'].join('');
```

Comment:
You can use template strings:
```suggestion
    const url = `/pack/xdsl/${opts.packName}/addressMove/offers`;
```